### PR TITLE
Add type for supportinternationalisationId

### PR DESCRIPTION
**Pull request by david-pepper**
Add a type for supportinternationalisationId to restrict acceptable values.

### CODE REVIEW

**Comment by andrewHEguardian** [comment]
it looks like there is a [README](https://github.com/guardian/support-frontend/tree/fc67babcbcb729a519c5d24ae1377b7e1816c490/support-frontend/assets/helpers/internationalisation) that needs updating

**Review by andrewHEguardian — APPROVED** [review]
this looks sensible to me 👍

File: support-frontend/assets/helpers/internationalisation/countryGroup.ts [hunk 1]
```diff
@@ -29,6 +29,15 @@ export type CountryGroupName =
 	| 'New Zealand'
 	| 'Canada';
 
+export type SupportInternationalisationId =
```
Comment (jamesgorrie) [inline support-frontend/assets/helpers/internationalisation/countryGroup.ts:32]:
```suggestion
export type InternationalisationId =
```

Just a thought - `Support` is not a hugely useful prefix.

**Review by jamesgorrie — APPROVED** [review]
Fabulous

File: support-frontend/assets/helpers/internationalisation/countryGroup.ts (hunk 1 above)
Comment (david-pepper) [reply support-frontend/assets/helpers/internationalisation/countryGroup.ts:32]: Thanks James - perhaps one for a future PR

**Pull request merged**